BOOTSTRAP.adoc

```diff
@@ -97,6 +97,17 @@ There are five steps to renaming a primitive:
 It is desirable for bootstraps to be easily repeatable, so you should commit
 changes after step 4.
 
+To remove a primitive:
+1. Start with a working build of the compiler eg `./configure && make world`
```

Comment:
The [rendered version](https://github.com/ocaml/ocaml/blob/b34836a28c5d53f3dd5f49c7d47e6eeb1480b3aa/BOOTSTRAP.adoc) looks off, I think you need to break a line before `1.` for the list to be parsed correctly.

Reply:
This [rendered version](https://github.com/ocaml/ocaml/blob/1fc0609d3faa35acb56d27a92bae3c21efafb982/BOOTSTRAP.adoc) is fixed. Apologies for the noise.